src/Parser/Mage/Fire/CombatLogParser.js

```diff
@@ -33,6 +34,7 @@ class CombatLogParser extends CoreCombatLogParser {
     cooldownThroughputTracker: CooldownThroughputTracker,
     damageDone: [DamageDone, { showStatistic: true }],
     cancelledCasts: CancelledCasts,
+    deathTracker: DeathTracker,
```

Comment:
How about enabling this in the core CombatLogParser so it's enabled for every spec?